x-pack/test/cloud_security_posture_functional/helpers.ts

```diff
@@ -5,23 +5,31 @@
  * 2.0.
  */
 
-const CLOUD_SECURITY_POSTURE_PACKAGE_NAME = 'cloud_security_posture';
+const CLOUD_SECURITY_PACKAGE_NAME = 'cloud_security_posture';
+const CLOUD_SECURITY_PACKAGE_VERSION = '1.2.5';
```

Comment:
@kfirpeled - since https://github.com/elastic/kibana/pull/148713 is merged, i think we can use any recent package and not just the ones before CSPM changes.

Reply:
you can check the logs and see if it doesn't fail to install the integration

Reply:
i did. but we agreed on `1.0.8` and i changed it in this PR, because it's not required anymore after ofir's fix.